### PR TITLE
Hide some fields in admin>orders>order details on phones

### PR DESCRIPTION
**Pull request by scottcwilson**
On the order details screen, when using a phone, hide these fields:
- all tax fields and model from the order table 
- customer notified and updated_by from the history table

### CODE REVIEW

**Comment by drbyte** [comment]
I can see the value of hiding the tax-related columns because those aren't material to the order-fulfillment process. (But they'd be important if one was making live edits to the order.)

However, the product model number is probably relevant for many stores. I'm not sure everyone would be happy with that exclusion.

**Comment by scottcwilson** [comment]
What about this: provide products model in a span underneath the name but with style .hidden-sm-up (i.e. it only displays on xs)?

**Comment by torvista** [comment]
>the product model number is probably relevant for many stores
For me, model number is absolutely non-negotiable with 100's of products with very similar names. So, all I touch has this in mind, from a self-interested point of view.

But, that's just me... I can't say for what proportion of users the model number is just wasted space, but if it is a "significant" %, perhaps it should be a global option "show model number" that would remove all display of it, columns etc. to have some sort of continuity between admin screens.

**Comment by scottcwilson** [comment]
I get it - but those product names probably also won't fit in a field as narrow as the current product name field on the admin orders screen.  So help me figure out a compromise, please. 

Bear in mind that this is on a phone only.  You can see everything if you go to your desktop computer or even your tablet.  To make it workable on a phone, we need to cut some things. 

What did you think of the suggestion of putting the model under the name on a mobile phone?

**Comment by scottcwilson** [comment]
With no changes, here's what it looks like: 
Screen Shot 2020-05-05 at 5 04 23 PM

**Comment by scottcwilson** [comment]
And here's what it would look like with this PR: 
Screen Shot 2020-05-05 at 5 12 40 PM

**Comment by torvista** [comment]
I think if you are looking at the order DETAILS page it's because you want the details. All of them.
Remember this is an Admin screen, so pretty is second to content.
As it is now it compresses well, the customer personal details take up far too much vertical space, but that's Bootstrap for you.
I wouldn't change anything, but then again I wouldn't consider managing my store on a phone either!

**Comment by torvista** [comment]
....I think the layout is wrong anyway, the product identification should come first (model-name) then the quantity etc..

File: admin/orders.php [hunk 1]
```diff
@@ -718,27 +718,31 @@ function couponpopupWindow(url) {
                         echo '</i></small></span>';
                       }
                     }
+                    // Mobile phones only
+                    echo '<span class="hidden-sm-up">';
```
Comment (drbyte) [inline admin/orders.php:722]: `hidden-sm-up` - Is that official? or accidentally works?  I don't see it in the docs

Comment (scottcwilson) [reply admin/orders.php:722]: Most official. 
https://v4-alpha.getbootstrap.com/layout/responsive-utilities/

Comment (drbyte) [reply admin/orders.php:722]: But ... we're currently using BS 3, not 4
Plus, the v4-alpha docs on that are quite different from the official v4 docs

Comment (scottcwilson) [reply admin/orders.php:722]: Two weeks ago when I did this, I could have sworn I saw this in the CSS.  Was it the whiskey?  Maybe.

Comment (barco57) [reply admin/orders.php:722]: Should be .hidden-sm for BS 3.4.1

Comment (barco57) [reply admin/orders.php:722]: https://getbootstrap.com/docs/3.4/css/#responsive-utilities

Comment (scottcwilson) [reply admin/orders.php:722]: No it's the opposite.  `hidden-sm-up` means only show on xs.

Comment (drbyte) [reply admin/orders.php:722]: I think all BS styles work that way: All apply from "specified size, and larger, until a larger-size rule is provided"

Comment (drbyte) [reply admin/orders.php:722]: Nope, not with .hidden-*

Comment (drbyte) [reply admin/orders.php:722]: Looks like `visible-xs-inline` is what's needed here.

**Pull request merged**